src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RizzComic.kt

```diff
@@ -110,14 +110,17 @@ class RizzComic : MangaThemesiaAlt(
         @SerialName("genre_id") val genres: String? = null,
     ) {
         val slug get() = title.trim().lowercase()
+            .replace("'", "")
             .replace(slugRegex, "-")
             .replace("-s-", "s-")
             .replace("-ll-", "ll-")
+            .replace(slugCleanupRegex, "")
 
         val genreIds get() = genres?.split(",")?.map(String::trim)
 
         companion object {
             private val slugRegex = Regex("""[^a-z0-9]+""")
+            private val slugCleanupRegex = Regex("""^-+|-+$""")
```

Comment:
Trim instead? You can set a custom char https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.text/trim.html